docs/source/tutorials/semantic_image_search.ipynb

```diff
@@ -0,0 +1,1222 @@
+   "id": "6a887a56-7bcb-4b7d-9f55-de4ccf13da9d",
+   "metadata": {},
+   "source": [
+    "And let's import the last few packages we'll need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7e8d6c1e-f074-4af4-af1d-6eedb29c9f17",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from pkg_resources import packaging\n",
+    "import torch\n",
+    "\n",
+    "device = \"cuda\" if torch.cuda.is_available() else \"cpu\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "725c5319-62c0-4630-9ad3-a8ebab46e0fb",
+   "metadata": {},
+   "source": [
+    "## Generating CLIP embeddings"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07f798be-b842-4730-8a2e-73617ad6f0d9",
+   "metadata": {},
+   "source": [
+    "To perform semantic search, we need to generate embeddings for the images in our dataset, as well as any text prompts we might want to search against. \n",
+    "\n",
+    "With FiftyOne’s `compute_embeddings()` method, we can generate embeddings for all of the images in our dataset in one fell swoop, storing them in an `embedding` field:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "94b578b4-8012-420c-aa87-f835cc32d493",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": []
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " 100% |███████████████| 5000/5000 [3.4m elapsed, 0s remaining, 24.6 samples/s]      \n"
+     ]
+    }
+   ],
+   "source": [
+    "dataset.compute_embeddings(\n",
+    "    model, \n",
+    "    embeddings_field=\"embedding\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe6e54e4-ae0f-48dc-ba24-6afc34f0ebc4",
+   "metadata": {},
+   "source": [
+    "This might take a few minutes, as computing embeddings from deep neural models is in general a relatively intensive task. If you plan to generate embeddings for many samples, best practice is to pre-compute them ahead of time, and if you intend to use these embeddings more than once, I suggest you persist this data by setting"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "4a44b077-efa9-4cb1-b494-95c222370e27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset.persistent = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d5b2aff-0d97-4ec4-87ff-548145ae8d4b",
+   "metadata": {},
+   "source": [
+    "If we’d like, we can also pair these embeddings with a dimensionality reduction technique to visualize the embeddings. Using the [FiftyOne Brain](https://voxel51.com/docs/fiftyone/user_guide/brain.html), we can do this with the `compute_visualization()` method:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "7a13dfa7-6515-4166-a866-7726bbd2f84a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
```

Comment:
I would just delete the output of this cell

Reply:
Sorry which cell is this? Can't see anything about the cell from this comment